cost/aws/reserved_instances/utilization/README.md

```diff
@@ -19,6 +19,12 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer